#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_nostrip.dpatch by Timur Birsh <taem@linukz.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Do not strip symbols

@DPATCH@

--- a/Makefile	2003-01-06 02:18:07.000000000 +0600
+++ b/Makefile	2009-08-17 14:00:21.000000000 +0600
@@ -32,7 +32,7 @@
 
 install: cd-discid
 	$(INSTALL) -d -m 755 $(bindir)
-	$(INSTALL) -s -m 755 -o 0 cd-discid $(bindir)
+	$(INSTALL) -m 755 -o 0 cd-discid $(bindir)
 	$(INSTALL) -d -m 755 $(mandir)
 	$(INSTALL) -m 644 -o 0 cd-discid.1 $(mandir)
 
